docs/features.rst

```diff
@@ -225,3 +225,26 @@ Features related to the type system
      - Not supported
      - However, an ``Input`` can receive data published by other *Connext DDS* applications that use FlatData.
 
+Loading Connext DDS Add-On Libraries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Connector* supports features that require the loading of additional *Connext DDS*
+libraries, such as
+`Monitoring <https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds_professional/users_manual/index.htm#users_manual/PartMonitoringLib.htm>`__
+and `Security Plugins <https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds_secure/getting_started_guide/index.html>`__.
+
+To use RTI Connext DDS add-ons you need an RTI Connext DDS installation. To
```

Comment:
Add something like this:

The Monitoring and Security plugins are configured in XML as described in the previous links.